src/Perpetuum/Services/ProductionEngine/Facilities/ProductionFacility.cs

```diff
@@ -255,7 +255,7 @@ protected void CreateSystemStorage()
 
         protected void RemoveStorage()
         {
-            Repository.DeleteTree(StorageEid);
+            Repository.DeleteTreeButNotRoot(StorageEid);
```

Comment:
This fixes #371 
On death/deconstruction the facility system-storages would be deleted as children on the parent, AND by deleting itself and the tree of items it may contain.
2 Delete calls, where the next fails and breaks the transaction.  
The base disappears from memory, but not from the DB and will reappear on restart!